apps/web-login/js/CallBack.js

```diff
@@ -0,0 +1,19 @@
+const naverOuath_ID = "Hw7RwBFOB7UjSufSEEjy";
+const callBack_URL = new URL(document.location.href);
```

Comment:
변수명의 규칙이 이상한 것 같습니다. 카멜케이스와 스네일 케이스가 왜 혼합되어있을까요?
```suggestion
const callBackUrl = new URL(document.location.href);
```

Reply:
추가로 파일명도 명확하지 않은 것 같습니다. 왜 파일명을 CallBack.js로 두셨을까요?

Reply:
#1 facebook이랑 google때 네이버 처럼 따로 callback.html이 있을 거 같아서 일부러 두었습니다

Reply:
facebook이랑 google  Oauth 로그인을 만들 때도 naver처럼 따로 callBcak.html이 필요할 거 같아서 따로 만들었습니다